charts/hivemq-platform-operator/values.yaml

```diff
@@ -109,6 +109,11 @@ podSecurityContext:
   enabled: false
```

Comment:
Please also change the above : `Holds pod-level security attributes and common container settings.` --> `Configures pod-level security for the HiveMQ Platform Operator pod.`